buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/converter.py

```diff
@@ -45,4 +83,19 @@ def get_metadata(self):
 
         # Get today date
         metadata["NWBFile"]["session_start_time"] = date
+
+        # Add additional NWBFile metadata
+        # NOTE: experimenters is specifid in the metadata.yml file
+        metadata["NWBFile"]["session_id"] = session_mat["session"]["general"]["name"]
+        metadata["NWBFile"]["notes"] = session_mat["session"]["general"]["notes"]
+
+        # Add Subject metadata
+        subject_metadata = session_mat["session"]["animal"]
+        metadata["Subject"]["subject_id"] = subject_metadata["name"]
+        metadata["Subject"]["sex"] = subject_metadata["sex"][0]
+        metadata["Subject"]["strain"] = subject_metadata["strain"]
+        metadata["Subject"]["genotype"] = subject_metadata["geneticLine"]
+
+        # NOTE: No weight information because there isn't any surgery metadata
```

Comment:
Always worth a check in their subject database: https://buzsakilab.com/wp/animals/